pkg/kubeapiserver/options/authentication.go

```diff
@@ -464,14 +460,13 @@ func (o *BuiltInAuthenticationOptions) ApplyTo(authInfo *genericapiserver.Authen
 		authInfo.APIAudiences = authenticator.Audiences{o.ServiceAccounts.Issuer}
```

Comment:
I just noticed the "`" quotes around the example in the service-account-issuer help make it render weirdly.

Drop the "`" quotes wrapping {service-account-issuer}/.well-known/openid-configuration above